Dynatrace/dynatrace-operator
# Fix json format of data-ingest injected file

# Description
Adds missing curly braces to data-ingest injection file.

## How can this be tested?
Run operator with data-ingest injection enabled (default).
Check if `/var/lib/dynatrace/enrichment/dt_metadata.json` is valid json.


## Checklist
~- [ ] Unit tests have been updated/added~
